### DISCUSSION TITLE
pimcore show colon in textfield without label

### DISCUSSION THREAD

**Discussion by jkdc** [pimcore/pimcore#13135] [General]
I have a class with some text input. some of them haven't Label, but pimcore is showing the colon. 
¿Does know any of you why is showing this colon?
![image](https://user-images.githubusercontent.com/12255305/189712713-98725c5e-96c4-4114-a991-24980356d212.png)

![image](https://user-images.githubusercontent.com/12255305/189712309-4d43afc7-042d-4613-9dd3-de44d3a43ce8.png)

**Comment by kingjia90** [comment]
Could it be that translation are messed up?
If you activate this checkbox, do you see `++:` ?
![image](https://user-images.githubusercontent.com/6014195/189715503-a2778790-85b1-47e2-bb3b-eb93684298ac.png)

**Reply by jkdc** [reply]
Hi, sorry for the delay. I've activated this checkbox but I can't see ++ in these places
![image](https://user-images.githubusercontent.com/12255305/191185159-dd9c913b-69ca-498d-9b98-81b0a80906d4.png)
Any ideas?

**Reply by dvesh3** [reply]
@jkdc please make sure the labels are not saved with a whitespace.

**Reply by jkdc** [reply]
@dvesh3 yes, I'm pretty sure that labels are totally empty
